phial/bot.py

```diff
@@ -327,6 +330,11 @@ def _execute_response(self,
         if isinstance(response, str):
             self.send_message(Response(text=response, channel=command.channel))
 
+        elif (isinstance(response, list) and
```

Comment:
It feels a bit wrong to me be sending `MessageAttachments` when they are not linked to a message. I feel like they should only be able to be sent as a property of the `Response` class